process/accounts/accountsProcessor.go

```diff
@@ -189,9 +194,17 @@ func (ap *accountsProcessor) getESDTInfo(accountESDT *data.AccountESDT) (*big.In
 	if accountESDT.TokenIdentifier == "" {
 		return nil, "", nil
 	}
+	if accountESDT.IsNFTOperation && accountESDT.NFTNonceString == "" {
+		return big.NewInt(0), "", nil
+	}
+
+	tokenKey := []byte(core.ElrondProtectedKeyPrefix + core.ESDTKeyIdentifier + accountESDT.TokenIdentifier)
+	if accountESDT.IsNFTOperation {
+		nonceBig, _ := big.NewInt(0).SetString(accountESDT.NFTNonceString, 10)
```

Comment:
for safety reasons, I would check if the `SetString` worked

Reply:
done.